static/app/utils/withIssueTags.tsx

```diff
@@ -45,47 +45,44 @@ function withIssueTags<Props extends WithIssueTagsProps>(
       teams: TeamStore.getAll(),
     });
 
-    const setAssigned = useCallback(
-      (newState: Partial<WrappedComponentState>) => {
-        setState(oldState => {
-          const usernames: string[] = newState.users
-            ? newState.users.map(getUsername)
-            : oldState.users.map(getUsername);
-
-          const teamnames: string[] = (newState.teams ? newState.teams : oldState.teams)
-            .filter(team => team.isMember)
-            .map(team => `#${team.slug}`);
-
-          const allAssigned = ['[me, none]', ...usernames, ...teamnames];
-          allAssigned.unshift('me');
-          usernames.unshift('me');
-
-          return {
-            ...oldState,
-            ...newState,
-            tags: {
-              ...oldState.tags,
-              ...newState.tags,
-              assigned: {
-                ...(newState.tags?.assigned ?? oldState.tags?.assigned ?? {}),
-                values: allAssigned,
-              },
-              bookmarks: {
-                ...(newState.tags?.bookmarks ?? oldState.tags?.bookmarks ?? {}),
-                values: usernames,
-              },
-              assigned_or_suggested: {
-                ...(newState.tags?.assigned_or_suggested ??
-                  oldState.tags.assigned_or_suggested ??
-                  {}),
-                values: allAssigned,
-              },
+    const setAssigned = useCallback((newState: Partial<WrappedComponentState>) => {
+      setState(oldState => {
+        const usernames: string[] = newState.users
+          ? newState.users.map(getUsername)
+          : oldState.users.map(getUsername);
+
+        const teamnames: string[] = (newState.teams ? newState.teams : oldState.teams)
+          .filter(team => team.isMember)
+          .map(team => `#${team.slug}`);
+
+        const allAssigned = ['[me, none]', ...usernames, ...teamnames];
+        allAssigned.unshift('me');
+        usernames.unshift('me');
+
+        return {
+          ...oldState,
+          ...newState,
+          tags: {
+            ...oldState.tags,
+            ...newState.tags,
+            assigned: {
+              ...(newState.tags?.assigned ?? oldState.tags?.assigned ?? {}),
+              values: allAssigned,
             },
-          };
-        });
-      },
-      [state]
-    );
+            bookmarks: {
+              ...(newState.tags?.bookmarks ?? oldState.tags?.bookmarks ?? {}),
+              values: usernames,
+            },
+            assigned_or_suggested: {
+              ...(newState.tags?.assigned_or_suggested ??
+                oldState.tags.assigned_or_suggested ??
+                {}),
+              values: allAssigned,
+            },
+          },
+        };
+      });
+    }, []);
```

Comment:
This was the only change, but it sadly triggered prettier to move everything by 1 line